packages/components/app/styles/components/tabs.scss

```diff
@@ -76,11 +77,12 @@
   cursor: pointer;
 
   // Expand click target area
-  &::after {
-    position: absolute;
-    content: "";
-    inset: 0;
-  }
+  // I don't think we need this anymore?
+  // &::after {
+  //   position: absolute;
+  //   content: "";
+  //   inset: 0;
+  // }
```

Comment:
It makes the button click target cover the full "tab" li including the padding and other space around the button in the li wrapper so I think it's still helpful.

Reply:
ok, I'll put it back in 👍